packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/MappingTestState.ts

```diff
@@ -84,6 +81,7 @@ import {
   PureClientVersion,
 } from '@finos/legend-graph';
 import { LambdaEditorState, TAB_SIZE } from '@finos/legend-application';
+import type { MappingTestResult } from '@finos/legend-graph/src/graphManager/action/execution/ExecutionResult';
```

Comment:
Need to ensure legend-graph exports `MappingTestResult`

Reply:
yeh, added it to the export list